README.md

```diff
@@ -7,46 +7,45 @@
 
 Interpret Community SDK
 =============================================================
+[InterpretML](https://github.com/interpretml/) is a machine learning interpretability package for training interpretable machine learning models and/or explaining blackbox AI systems. InterpretML that consists of the following repositories:
```

Comment:
Calling InterpretML a package might be confusing because it's not something that a user can actually install. Instead, it's more like a suite of packages, not sure what you want to call that.